### ISSUE TITLE
I'm trying to use RTSP, a CCTV communication protocol, with yolo8 images.

### ISSUE THREAD

**Issue by rurusungoa** [ultralytics/ultralytics#9900]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

hello
I'm trying to use RTSP, a CCTV communication protocol, with yolo8 images.
Should I use tcp as the protocol?
Should I use the rtsp protocol?

When the communication protocol is Reverse, is the communication method TCP?

I would like to inquire whether you used the TCP protocol when doing the project.

    port: 558 
    protocol: TCP ==> Should I change it to rtsp?
    targetPort: 558

   bind: 558
   mode tcp => Should I change it to rtsp?
   option tcplog =>Should I change it to rtsplog?

**Comment by glenn-jocher** [comment]
Hello! 😊

For integrating RTSP streams with YOLOv8 for CCTV footage analysis, you indeed need to use the RTSP protocol in your configurations. Typically, RTSP streams operate over TCP/IP, but when specifying the protocol for YOLO inference, you should directly use the RTSP URI format.

Here's a simple snippet to use an RTSP stream with YOLOv8:

```python
from ultralytics import YOLO

# Initialize YOLO model
model = YOLO('yolov8n.pt')

# Specify your RTSP stream URL
rtsp_url = 'rtsp://your_camera_ip:558/your_stream_path'

# Run inference on the RTSP stream
results = model(rtsp_url, stream=True)

# Process your results here
for result in results:
    result.show()  # Display the frame with detections
```

Remember to replace `'rtsp://your_camera_ip:558/your_stream_path'` with your actual RTSP stream URL.

In summary, for RTSP:
- Use the `rtsp` protocol in your URI.
- The connection will indeed utilize TCP behind the scenes, but you specify the protocol as `rtsp` in your URL.

Hope this clears things up! Let us know if you have any more questions.
